Apsitvarkom.UnitTests/DataAccess/PollutedLocationDatabaseRepositoryTests.cs

```diff
@@ -58,17 +41,19 @@ public async Task GetAllAsync_DbContextHasOneInstance_SingleInstanceReturnedWith
         var pollutedLocationInstance = new PollutedLocation 
         { 
             Id = Guid.NewGuid(),
-            Coordinates = new Coordinates { Latitude = 47.12, Longitude = -41.1251 }, 
+            Location = {
```

Comment:
this seems to be the only place where we have the curly bracket in the same line as Location.